DataFormats/simulation/include/SimulationDataFormat/MCTruthContainer.h

```diff
@@ -79,15 +79,19 @@ class MCTruthContainer
 
   // get individual "view" container for a given data index
   // the caller can do modifications on this view (such as sorting)
-  gsl::span<TruthElement> getLabels(int dataindex) {
-    if(dataindex >= getIndexedSize()) return gsl::span<TruthElement>();
+  gsl::span<TruthElement> getLabels(uint dataindex)
+  {
+    if (dataindex >= getIndexedSize())
+      return gsl::span<TruthElement>();
     return gsl::span<TruthElement>(&mTruthArray[mHeaderArray[dataindex].index], getSize(dataindex));
   }
 
   // get individual const "view" container for a given data index
   // the caller can't do modifications on this view
-  gsl::span<const TruthElement> getLabels(int dataindex) const {
-    if(dataindex >= getIndexedSize()) return gsl::span<const TruthElement>();
+  gsl::span<const TruthElement> getLabels(uint dataindex) const
+  {
+    if (dataindex >= getIndexedSize())
+      return gsl::span<const TruthElement>();
```

Comment:
personally I would add braces around the code line when clang-format moves a single statement to the next line

Reply:
Agreed (it is also in our coding guidelines).